src/data/LowerLayout/aboutCopy.js

```diff
@@ -0,0 +1,21 @@
+const copy = {
```

Comment:
I would actually just put this straight up in a `JSON` file, since you can import and destructure JSON files in Webpack 3+. This also tells someone looking at the code that this is only static values and there is no JavaScript logic in the file.